plane/source/docs/tuning-cruise.rst

```diff
@@ -1,37 +1,43 @@
 
-While the autopilot will operate correctly with minor offsets to the calibrated "level" attitude, optimum performance will occur if this adjusted to match that pitch attitude. This can be done by examining the average ``ATT.pitch`` value during level altitude cruise from dataflash logs, or visually if using an OSD or GCS, to adjust the horizon line level using slight adjustments to the :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` parameter value. This also will allow STABILIZE and FBWA modes to hold altitude at the same throttle level as used in the other modes.
+While the autopilot will operate correctly with minor offsets to the calibrated "level" attitude, optimum performance will occur if this is adjusted to match the pitch attitude that the autopilot tends to use in automatic throttle modes. This can be done by examining the average ``ATT.pitch`` value during level altitude flight from dataflash logs or in-flight on an artificial horizon (e.g., GCS or OSD HUD). Make slight adjustments to the :ref:`TRIM_PITCH_CD<TRIM_PITCH_CD>` parameter value based on the value seen on the artificial horizon. Making these adjustments also will allow STABILIZE and FBWA modes to hold altitude at the same throttle level as used in the other modes.
 
-.. note:: It is important that the :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>` parameter be set correctly. This parameter will add "nose-down" trim when the throttle is lowered in pilot throttle controlled, stabilized modes, such as FBWA, to prevent the autopilot from holding the nose up as the plane slows down and potentially stalling. This can be tested at altitude in FBWA mode by moving the throttle to idle and checking that there is sufficient airspeed in a turn to avoid stalling (be prepared to do a stall recovery, which might occur). Increase the value of :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>`, if necessary.
+.. image:: ../../../images/AOA.jpg
 
-Adjusting CRUISE Mode Speed
-===========================
+The autopilot's priority is to obtain the correct combination of elevator and throttle to maintain constant altitude flight. How the autopilot does this is detailed in :ref:`tecs-total-energy-control-system-for-speed-height-tuning-guide`.
+
+The :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>` parameter will add "nose-down" trim when the throttle stick is lowered in pilot throttle controlled and stabilized modes, such as FBWA, to prevent the autopilot from holding the nose up as the plane slows down and potentially causing a stall. This can be tested, at altitude, in FBWA mode by moving the throttle to idle and checking that there is sufficient airspeed in a turn to avoid stalling. Be prepared to recover from a stall! Increase the value of :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>`, if necessary.
+
+.. tip:: For small planes or gliders, it is important that :ref:`STAB_PITCH_DOWN<STAB_PITCH_DOWN>` be set correctly. It might be desireable to increase STAB_PITCH_DOWN more than the default value of 2 degrees.
```

Comment:
redundant to above paragraph

Reply:
and here...line 37

Reply:
The parameter can be important to many aircraft. As you said, it is particularly important for smaller aircraft or low Reynolds number aircraft so I called those out in an admonition.

Reply:
the tip is a redudant restatement of the paragraph above...the sentence above it states essentially what the tip says

Reply:
The paragraph is describing the purpose of the parameter, the cases where it comes into play, where it is useful, and how to test it. The tip is calling out that it is more important for smaller birds and suggests in these cases that changes might be desirable.